labrad/test/test_eval_lr_data.py

```diff
@@ -0,0 +1,22 @@
+import pytest
+
+
+import labrad.types as T
+from labrad.units import Complex, Value
+
+
+def test_eval_value_with_none_for_units():
+    assert T.evalLRData("""Value(1.0, None)""") == Value(1.0, '')
```

Comment:
I think when I first wrote that I was anticipating quotes in the value, but of course it's using `None` not `''` for units, so the triple quotes are not needed. I'll change it.